pom.xml

```diff
@@ -184,6 +184,12 @@
             <version>1.3.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito.kotlin</groupId>
+            <artifactId>mockito-kotlin</artifactId>
+            <version>4.0.0</version>
```

Comment:
You should not need the version here if you update the parent pom at the top of the file as this should be pulled from there to keep dependencies more consistent between projects.